alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/LabelValueSelectOptions.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.descriptor.config.field;
+
+import java.util.List;
+
+public class LabelValueSelectOptions extends FieldOptions<LabelValueSelectOption> {
```

Comment:
Is there a reason we are keeping this around? It seems like we have been changing the endpoints to return their own custom versions of FieldOptions. EX: ProviderProjectOptions.
It seems like the code is split between using this class as a general Option vs custom implementations of FieldOptions. Is the plan to use the class to replace those custom classes? or to replace this by creating more custom classes?

Reply:
If the other endpoints are returning FieldOptions<?> then I will change the usage to be consistent with what has been done.

Reply:
Look at most of the custom endpoints. 80% or more use this.

Reply:
Is that the distinction? The custom endpoints will use this class, while the other endpoints will create custom FieldOptions for the swagger?
I just want to figure out what the difference is

Reply:
The reason we have the specific types is because the response would be ActionResponse<FieldOptions<LabelValueSelectOption>> for the method.  In other places where I had a `ActionResponse<List<SomeObject>>` we wanted an object that wraps a list and return something like this `ActionResponse<WrappedListObject>`.

See this PR: 
https://github.com/blackducksoftware/blackduck-alert/pull/1158

Since that was the preference of the team I created the following list wrapper objects:
LabelValueSelectOptions
ProviderProjectOptions
EmailAddressOptions
NotificationFilterModelOptions
ProviderProjectOptions

This was done in this PR: 
https://github.com/blackducksoftware/blackduck-alert/pull/1160

The implementation was the same for each of these objects only the generic type was different.  So I created FieldOptions so I wouldn't write the same code over and over again.